src/main/java/seedu/realodex/ui/UiManager.java

```diff
@@ -65,7 +65,7 @@ void showAlertDialogAndWait(Alert.AlertType type, String title, String headerTex
     private static void showAlertDialogAndWait(Stage owner, AlertType type, String title, String headerText,
                                                String contentText) {
         final Alert alert = new Alert(type);
-        alert.getDialogPane().getStylesheets().add("view/DarkTheme.css");
+        alert.getDialogPane().getStylesheets().add("view/RealodexTheme.css");
```

Comment:
No idea how to test this, gpt is not useful, worse case rename it back to DarkTheme lol